features/embedded-wallets/ephemeral-embedded-wallet.ts

```diff
@@ -0,0 +1,55 @@
+import { IEmbeddedWallet } from "./embedded-wallet.interface";
+import { Client } from "@xmtp/react-native-sdk";
+import {
+  generatePrivateKey,
+  LocalAccount,
+  privateKeyToAccount,
+} from "viem/accounts";
+import {
+  copyDatabasesToTemporaryDirectory,
+  createTemporaryDirectory,
+} from "@/utils/fileSystem";
+import { getDbEncryptionKey } from "@/utils/keychain/helpers";
+import { getInboxId } from "@/utils/xmtpRN/signIn";
+import { viemSignerToXmtpSigner } from "@/utils/xmtpRN/signer";
+
+export class EphemeralEmbeddedWallet implements IEmbeddedWallet {
+  account: LocalAccount<"privateKey"> | null = null;
+  xmtpClient: Client | null = null;
+
+  constructor(public readonly address: string) {}
```

Comment:
Michael started using `ethAddress` or something like that and I lke that I think. Instead of always using `address`